src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -217,5 +217,7 @@ public static class ScriptConstants
 
         public static readonly string FunctionMetadataDirectTypeKey = "DirectType";
         public static readonly string LiveLogsSessionAIKey = "#AzFuncLiveLogsSessionId";
+
+        public static readonly string FunctionsHostingConfigSectionName = " FunctionsHostingConfig";
```

Comment:
Think you have an extra whitespace here:
```suggestion
        public static readonly string FunctionsHostingConfigSectionName = "FunctionsHostingConfig";
```